pkg/bootstrap/config.go

```diff
@@ -79,15 +79,28 @@ func (cfg Config) toTemplateParams() (map[string]interface{}, error) {
 		xdsType = "DELTA_GRPC"
 	}
 
+	// Add the meshScope
+	meshScope := "default"
+	if scope, found := cfg.Metadata.Labels["gke.io/mesh"]; found {
```

Comment:
would prefer to just pass `cfg.Metadata` then in the template do this. To keep the gcp stuff in the gcp bootstrap file.

Reply:
sent Metadata.Labels instead as i was not able to make Metadata work. This change does move the gcp stuff in the gcp bootstrap file. LMK WYT.